src/Utils/Notifications.js

```
@@ -74,27 +40,26 @@ const notifyError = (error) => {
-  notifyStack.close();
+  toast.dismiss();
 };
 
 /** Success message handler */
 export const Success = ({ msg }) => {
-  notify(msg, "success");
+  toast.success(msg, { duration: 3000 });
 };
 
 /** Error message handler */
 export const Error = ({ msg }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)